src/components/ATATSummaryCard.vue

```diff
@@ -146,13 +161,28 @@ export default class ATATSummaryCard extends VCard {
     return true;
   }
 
-  @Emit()
   private leftButtonClicked(card: ATATSummaryCardItem) {
-    return true;
+    let id = card.title;
```

Comment:
Let's emit this back to the parent either using propsync or @Emit, as we are keeping this card as generic as possible to be re-used.